### PR TITLE
Update 'What's new' section on /server

### LINKED ISSUE #8501: [20.10 launch] Update 'What's new' section on /server
[copy doc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit#)

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Copy docs [updates](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit#)

## QA

- Click on the demo https://ubuntu-com-8546.demos.haus/server and make sure the copy updates are correct.

## Issue / Card

Fixes #8501

### CODE REVIEW

File: templates/server/index.html [hunk 1]
```diff
@@ -39,6 +39,29 @@ <h1>Scale out with Ubuntu Server</h1>
   </div>
 </div>
 
+<div class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2>
+      What&rsquo;s new in {{ releases.latest }} LTS
+    </h2>
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Supported by Canonical until July 2021</li>
```
Comment (pmahnke) [inline templates/server/index.html:48]:
```suggestion
      <li class="p-list__item is-ticked">Supported by Canonical until {{ releases.latest.eol }}</li>
```

File: templates/server/index.html [hunk 2]
```diff
@@ -39,6 +39,29 @@ <h1>Scale out with Ubuntu Server</h1>
...
+      <li class="p-list__item is-ticked">Runs on all major architectures - x86-64, ARM v7, ARM64, POWER8, POWER9, IBM s390x (LinuxONE) and RISC-V</li>
+      <li class="p-list__item is-ticked">Nftables-based backend for ufw for improved performance and easier administration</li>
+      <li class="p-list__item is-ticked">Support for OVS schema modelling in Netplan for an advanced networking stack configuration</li>
+      <li class="p-list__item is-ticked">Native configuration for WireGuard VPN in Netplan</li>
+      <li class="p-list__item is-ticked">The latest long-term Linux 5.8 kernel for the latest hardware and security updates</li>
+      <li class="p-list__item is-ticked">Updates to QEMU (5.0), libvirt (6.6), PHP (7.4.9), Apache2, GCC (1.189), Python (3.8.6), Bind9 (9.16.6)</li>
```
Comment (pmahnke) [inline templates/server/index.html:54]: I have asked Rick for the Apache2 version

File: templates/server/index.html [hunk 3]
```diff
@@ -39,6 +39,29 @@ <h1>Scale out with Ubuntu Server</h1>
   </div>
 </div>
 
+<div class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2>
+      What&rsquo;s new in {{ releases.latest }} LTS
```
Comment (pmahnke) [inline templates/server/index.html:45]:
```suggestion
      What&rsquo;s new in {{ releases.latest.full_version }} LTS
```
you are outputting the entire yaml string

**Review by pmahnke — CHANGES_REQUESTED** [review]
some comments inline + we need the apache2 version.

File: templates/server/index.html (hunk 3 above)
Comment (carkod) [reply templates/server/index.html:45]: Is there a reason why we are writing full version here and short version in the previous one?

File: templates/server/index.html (hunk 1 above)
Comment (carkod) [reply templates/server/index.html]: Done. Thanks for that, didn't notice.

File: templates/server/index.html (hunk 2 above)
Comment (carkod) [reply templates/server/index.html]: Done

File: templates/server/index.html (hunk 3 above)
Comment (pmahnke) [reply templates/server/index.html]: there is _almost_ never a dot release of a latest, if there was, it wouldn't change the features, where an LTS might change more.

**Review by pmahnke — APPROVED** [review]
+1

**Pull request merged**